tar-changes

```diff
@@ -121,9 +132,12 @@ def main():
     osdist = guess.osdist()
 
     # "ceph-3.2-rhel-patches"
-    remote_patches_branch = guess.patches_branch(branch, pkg=name,
-                                                 osdist=osdist)
-    patches_branch = remote_patches_branch.partition('/')[2]
+    if not args.patches_branch:
```

Comment:
I think it would be easier to read this code at a glance if we avoided "`If not <false condition>`", by swapping the order of these conditions:
```python
if args.patches_branch:
    patches_branch = args.patches_branch
else:
    remote_patches_branch = guess.patches_branch(branch, pkg=name,
                                                 osdist=osdist)
    patches_branch = remote_patches_branch.partition('/')[2]
```

What do you think?

Reply:
Sounds good to me, I think I've got all this fixed now.